Models/AssemblyContainer.cs

```diff
@@ -1,8 +1,20 @@
-﻿namespace Hast.Remote.Bridge.Models
+using System;
+
+namespace Hast.Remote.Bridge.Models
 {
     public class AssemblyContainer
     {
         public string Name { get; set; }
-        public byte[] FileContent { get; set; }
+
+        private byte[] _fileContent;

```

Comment:
Aha, erre nem gondoltam. Átalakítottam így és mivel mindig `MemoryStream` használja fel azt kiemeltem metódusba.
``` csharp
// Using a property with the [JsonProperty] attribute makes it serializable even when it's otherwise inaccessible.
[JsonProperty]
private byte[] FileContent { get; set; }
```